website/docs/r/event_orchestration_global.html.markdown

```diff
@@ -0,0 +1,159 @@
+        # Id of the next set
+        route_to = "step-two"
+      }
+    }
+  }
+  set {
+    id = "step-two"
+    rule {
+      label = "Drop events that are marked as no-op"
+      condition {
+        expression = "event.summary matches 'no-op'"
+      }
+      actions {
+        drop_event = true
+      }
+    }
+    rule {
+      label = "If there's something wrong on the replica, then mark the alert as a warning"
+      condition {
+        expression = "event.custom_details.hostname matches part 'replica'"
+      }
+      actions {
+        severity = "warning"
+      }
+    }
+    rule {
+      label = "Otherwise, set the incident to P1 and run a diagnostic"
+      actions {
+        priority = data.pagerduty_priority.p1.id
+        automation_action {
+          name = "db-diagnostic"
+          url = "https://example.com/run-diagnostic"
+          auto_send = true
+        }
+      }
+    }
+  }
+  catch_all {
+    actions { }
+  }
+}
+```
+## Argument Reference
+
+The following arguments are supported:
+
+* `event_orchestration` - (Required) ID of the Event Orchestration to which this Global Orchestration belongs to.
+* `set` - (Required) A Global Orchestration must contain at least a "start" set, but can contain any number of additional sets that are routed to by other rules to form a directional graph.
+* `catch_all` - (Required) the `catch_all` actions will be applied if an Event reaches the end of any set without matching any rules in that set.
+
+### Set (`set`) supports the following:
+* `id` - (Required) The ID of this set of rules. Rules in other sets can route events into this set using the rule's `route_to` property.
+* `rule` - (Optional) The Global Orchestration evaluates Events against these Rules, one at a time, and applies all the actions for first rule it finds where the event matches the rule's conditions. If no rules are provided as part of Terraform configuration, the API returns empty list of rules.
+
+### Rule (`rule`) supports the following:
+* `label` - (Optional) A description of this rule's purpose.
+* `condition` - (Optional) Each of these conditions is evaluated to check if an event matches this rule. The rule is considered a match if any of these conditions match. If none are provided, the event will `always` match against the rule.
+* `actions` - (Required) Actions that will be taken to change the resulting alert and incident, when an event matches this rule.
+* `disabled` - (Optional) Indicates whether the rule is disabled and would therefore not be evaluated.
+
+### Condition (`condition`) supports the following:
+* `expression`- (Required) A [PCL condition](https://developer.pagerduty.com/docs/ZG9jOjM1NTE0MDc0-pcl-overview) string.
+
+### Actions (`actions`) supports the following:
+* `route_to` - (Optional) The ID of a Set from this Global Orchestration whose rules you also want to use with event that match this rule.
```

Comment:
I know the same wording is used in Service and Unrouted docs but should this say "**events** that match" (plural)?
```suggestion
* `route_to` - (Optional) The ID of a Set from this Global Orchestration whose rules you also want to use with events that match this rule.
```